homeassistant/components/homekit_controller/climate.py

```diff
@@ -209,6 +246,7 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
             )
         await self.async_put_characteristics(
             {
+                CharacteristicsTypes.ACTIVE: ActivationStateValues.ACTIVE,
```

Comment:
There is a separate PR with just this fix in which will be going out in a bug fix patch release, you'll need to rebase on `dev` when that lands (#81868).